### PR TITLE
Revert to BouncyCastle for .NET Standard 2.0 support

### PR DESCRIPTION
**Pull request by danielgindi**
This solves #44 - without multiple targets, just BouncyCastle as suggested in #45.

### CODE REVIEW

File: CorePush/CorePush.csproj [hunk 1]
```diff
@@ -39,6 +39,8 @@ Previous Versions:
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="System.Security.Cryptography.Cng" Version="5.0.0" />
```
Comment (andrei-m-code) [inline CorePush/CorePush.csproj:42]: Can this be deleted?

Comment (andrei-m-code) [reply CorePush/CorePush.csproj:42]: Probably no but please confirm.

Comment (danielgindi) [reply CorePush/CorePush.csproj]: Yeah, it works :-)

Comment (danielgindi) [reply CorePush/CorePush.csproj]: @andrei-m-code can you merge and publish a version?

Also there are a few other small PRs there.

**Comment by andrei-m-code** [comment]
@danielgindi all merged and new version 3.0.2 is going live as soon as nuget server processes it. Thank you!!!

**Pull request merged**